### ISSUE TITLE
Reports success when the image name was misspelled

### ISSUE THREAD

**Issue by Leonidas-from-XIV** [issuu/sure-deploy#6]
Internal report from @haljin.

Interesting how that would be failing, since I don't actually know how Docker Stack behaves when the image one is trying to run does not exist. Will investigate.

**Comment by Leonidas-from-XIV** [comment]
After a bit of digging I realized this is due to sure-deploy setting `completed` when no `UpdateStatus` was returned in the response. Unfortunately the `UpdateStatus` is *never* included when the stack is created for the first time (https://github.com/moby/moby/issues/28012) so there is a choice between always failing the first deployment (giving the impression that sure-deploy is not reliable) or always succeeding the first deploy (giving a false impression of correct deployment). A classic catch-22 if I have ever seen one.

Need to investigate whether there is a way to get the status in a different way.

**Comment by yangm97** [comment]
> Unfortunately the `UpdateStatus` is _never_ included when the stack is created for the first time

A workaround for getting the real status would be deploying new services twice.

**Comment by Leonidas-from-XIV** [comment]
Yes, but since it is not `sure-deploy` doing the deployment (to avoid creating a weird "wrapper" around `docker`), it is just ensuring the deployment succeeded or not, that's not really possible in that form.

And also a bit of a weird thing to do to begin with. It would be better if `docker` just provided a way to see what happened with the deployment.
